### PR TITLE
Add basic css and navigation bar

### PR DESCRIPTION
**Pull request by duz-sg**

### CODE REVIEW

**Comment by ethanysc** [comment]
Bootstrap setup should be written in main.hbs with the {{{body}}} tag, the rest of the hbs files should not have body tags in them

**Comment by ethanysc** [comment]
Also are we placing tab functions on the nav bars? I'm not sure how our kiosk software would benefit from a nav bar

**Comment by tymc7** [comment]
I'll pull down the branch and update

**Comment by tymc7** [comment]
@duz-sg if you want to add more updates, be sure to do a `git pull` before pushing to grab the updates I made

**Comment by ethanysc** [comment]
Added bootstrap to check-in screens.
Approve but don't merge yet, will work on the rest of the hbs tomorrow, thanks!

File: config.template.js [hunk 1]
```diff
@@ -1,13 +1,8 @@
-// copy this to your config file
-// connect to phpmyadmin: https://tools.engr.oregonstate.edu/phpMyAdmin/index.php
-// With your VPN connected, use below user/password to login
-// Server choice: classmysql
-// New table names should start with a_, like a_patients, so they are on the top of the list
-// and distinguishable with the tables I used in cs340
+// USE THIS CREATE YOUR CONFIG FILE
```
Comment (tymc7) [inline config.template.js:1]: Can you remove this file from the PR, I've updated it so that it fits a new configuration

**Review by tymc7 — COMMENTED** [review]
LGTM - Just remove the `config.template.js` from the commits and we should be good to go

File: config.template.js (hunk 1 above)
Comment (duz-sg) [reply config.template.js:1]: Not sure how to remove the file from PR, is it OK to just overwrite with the new file?

**Comment by tymc7** [comment]
Need @ethanysc to merge since I pushed changes to this PR

**Pull request merged**